radixdlt-java/src/main/java/com/radixdlt/client/core/BootstrapByTrustedNode.java

```diff
@@ -46,7 +48,11 @@ public BootstrapByTrustedNode(Collection<RadixNode> trustedNodes) {
 				throw new IllegalStateException("Could not retrieve universe configuration.", e);
 			}
 
-			return Serialize.getInstance().fromJson(universeJson, RadixUniverseConfig.class);
+			try {
+				return Serialize.getInstance().fromJson(universeJson, RadixUniverseConfig.class);
```

Comment:
Why does `SerializationException` require handling now? It clutters our code in many places.

Reply:
Good question! The serialization implementation in Java lib and Engine differed! @clstrfsck any input? Should I make `fromJson` non throwing? I understand @flotothemoon s point of cluttering the code. However, I'm very much for propagating errors and letting them float up to as high level as possible, rather than crashing "deep down"/low level.

Reply:
Hmm.  I think the difference here is the definition of SerializationException.  In radixdlt-java, SerializationException extends RuntimeException, but in radix-engine-library it extends IOException.

I really think the least likely to introduce bugs option here is to have SerializationException extend IOException.  Having it extend RuntimeException has the potential to introduce issues in the engine where catching IOException expects to catch SerializationException.

Reply:
@clstrfsck I.e. keeping `SerializationException` as it currently is implemented in this repo... 
So that means that I do need to keep the catch and crash (by throwing `IllegalStateException`) above, or do some other change...? Can the `Suppliers.memoize` be throwing?? 

Sounds like @flotothemoon and @clstrfsck need to agree on a strategy :)

Reply:
I thought we had agreed on throwing a checked (i.e. must-handle-in-some-way) exception for deserialization since the caller must expect this to go wrong. However, below an IllegalStateException is used.

Reply:
Yeah going to be changed in coming PR.